### PR TITLE
Splitting HUD code out of inventory slot rewrite.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
- Cleans up some HUD logic.
- Improves hand slot generation/ordering.
- Fixes issues with nymph HUD offsets.
- Expands nymph UI icons.

## Why and what will this PR improve
Nicer HUD code, prep for inventory slot rewrite PR.

## Authorship
Myself.

## Changelog
Nothing player-facing.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
By 'improves' it currently means 'only gives you one hand'

**Comment by MistakeNot4892** [comment]
All appears to be working now other than an issue identified with nymph UI that I've raised a ticket for.

File: code/modules/mob/living/inventory.dm [hunk 1]
```diff
@@ -58,6 +59,16 @@
 		if(istype(I))
 			I.on_active_hand()
 
+// Defer proc for the sake of delimbing root limbs with multiple graspers (serpentid)
+/mob/living/proc/queue_hand_rebuild()
+	set waitfor = FALSE
+	if(!pending_hand_rebuild)
+		pending_hand_rebuild = TRUE
+		sleep(1)
```
Comment (out-of-phaze) [inline code/modules/mob/living/inventory.dm:67]: Not a huge fan of this but until we get a queue/unique-queue subsystem to reuse, it'll do.

**Review by out-of-phaze — APPROVED** [review]
Will probably lead to conflicts with my nymph UI fixes when stable is merged into dev, but that shouldn't be too hard to resolve.

**Pull request merged**